back/engines/commercial/multi_tenancy/lib/tasks/core/add_last_survey_pages.rake

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+namespace :cl2back do
+  desc 'Adds a new post-submission page to all existing surveys with correct logic'
```

Comment:
Suggest you move this to the `:single_use` namespace in `back/lib/tasks/single_use`, giving it a date file prefix like the others in the folder. Keeps it more obviously separate from the other tasks that can be run more than once